### PR TITLE
Follow-up for initial website version

### PR DESCRIPTION
**Pull request by chrismayer**
This adds some adaptations for the initial Wegue website version:

  - Set Docsify base path
  - Change website title
  - Put QGIS2Wegue in a separate menu entry
  - Add more information about QGIS2Wegue
  - Hint for currently missing live demo

Please review @JakobMiksch (and anyone else of course)

### CODE REVIEW

File: docs/qgis_plugin.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
-# QGIS Plugin
+# QGIS2Wegue QGIS Plugin
 
-Adding many layers to a configuration file can be laborious, this is where `QGIS2Wegue` ([GitHub Repository](https://github.com/meggsimum/qgis2wegue)) comes in handy. It converts all layers of a QGIS project into a valid Wegue configuration. This works for `WMS`, `XYZ`, `KML`, `GeoJSON`, `WFS`.
+Configuring your Wegue application by hand can be laborious. This is where QGIS plugin `QGIS2Wegue` ([GitHub Repository](https://github.com/meggsimum/qgis2wegue)) comes in handy. It derives a valid Wegue application configuration from your QGIS project:
+
+  - All layers of a QGIS project are converted into a valid Wegue layer configuration. This works for `WMS`, `XYZ`, `KML`, `GeoJSON`, `WFS`
+  - The map extent is taken over to the configuration
+  - Textual elements, like title and footer can be set in a GUI
+  - The modules, which should be included in your Wegue application, be easily be added / removed via GUI
```
Comment (justb4) [inline docs/qgis_plugin.md:9]: Editorial: 
Modules to be included in your Wegue application can be easily added/removed via the GUI

**Comment by justb4** [comment]
Looks further good to me!

File: docs/qgis_plugin.md (hunk 1 above)
Comment (JakobMiksch) [reply docs/qgis_plugin.md:9]: yes, that is a better phrasing. 
@chrismayer could you replace this sentence with Just's suggestion?
-->  `Modules to be included in your Wegue application can be easily added/removed via the GUI`

**Review by JakobMiksch — COMMENTED** [review]
The changes look good to me. Only the suggested change by Just could be done.  See inline comment.

**Comment by chrismayer** [comment]
Thanks for your reviews @justb4 and @JakobMiksch! I added the suggested textual change. I'll merge now.

**Pull request merged**